theme/acquia_lift.wizard.theme.inc

```diff
@@ -45,6 +46,9 @@ function theme_acquia_lift_card($variables) {
   }
   $html .= '</div>';
   $html .= '<div class="el-card__content">';
+  if (!empty($card_flag)) {
```

Comment:
Be aware that this markup is in the document flow, so it will cause items to shift to accommodate the space it takes up. 

There is also an `is-active` class that could be used for dragons. Before this PR, it was being used for an expanded card and it was more intended as a "this is currently the piece to focus on" and there would only be one at a time. The cards now use `is-compact` for when the card is ... well ...  compact.